test/reducers/navigator.spec.js

```diff
@@ -56,4 +56,18 @@ describe('navigate reducer', () => {
     expect(state.stack.count()).to.be.equal(2);
     expect(state.stack.first()).to.be.equal('test');
   });
+
+  it('expect REPLACE_AT_INDEX to replace the route at the index passed', () => {
```

Comment:
Dont you think this is unecessary, because if this happens it will throw and the test will not pass.